core/controllers/profile_test.py

```diff
@@ -823,6 +823,8 @@ def test_export_account_handler(self):
         user_settings.last_agreed_to_terms = self.GENERIC_DATE
         user_settings.last_logged_in = self.GENERIC_DATE
         user_settings.validate()
+        # TODO(#10178): Remove gae_id attribute from UserSettingsModel below
```

Comment:
Just a note: you don't need to add all these TODOs to the codebase. When gae_id is removed, the tests will fail, and fixing the tests will remove these instances. Just one TODO in the storage layer is sufficient.